php/libraries/Project.class.inc

```diff
@@ -265,33 +268,18 @@ class Project
     }
 
     /**
-     * Update the Alias field in the databse for this Project.
+     * Update the Alias field in the database for this Project.
      *
-     * @param ?string $alias The new alias. Uses class property when null.
+     * @param ?string $projectAlias The new alias. Uses class property when null.
      *
      * @return void
      */
-    public function updateAlias(?string $alias = null): void
+    public function updateAlias(?string $projectAlias = null): void
```

Comment:
Alias is non-nullable in the DB. Maybe we should keep this a string?
```suggestion
    public function updateAlias(string $projectAlias = ''): void
```

Reply:
I had left that as is from your PR and adapted the other update functions to behave similarly.

since one validation is already being done in the set function, I would say we can add the non null check to the set function and the create new function maybe ?

Reply:
I think it makes sense to type hint as `string` so that it can't be null. Then we can use empty strings where we're using `null` and leave the empty() checks.

The types in the class should agree with the types in the DB or else we're inviting bugs.